core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+
+  public function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminTitle() {
+    return t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminPreview() {
+    return t('Displays a news feed from BackdropCMS.org.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent() {
+    $feed = backdrop_http_request('https://backdropcms.org/api/notifications');
```

Comment:
We could save the URL for the feed as a config value, in `layout.settings.json` so that we can have proper tests.

Reply:
I don't think `layout.settings.json` is the right place for it, as that's where we store settings for layouts (not for individual blocks). Why do we need it in config for tests? Could we use a constant instead?

Reply:
Yeah, `dashboard.settings.json` would be a better place. But I agree that we should have this in config. We do the same thing for update module, that allows us to use a different location for the XML feed (such as a local copy). It might also be a good idea to be able to disable this feed entirely during test runs so we're not hitting backdropcms.org on every test.